README.md

```diff
@@ -105,17 +105,36 @@ If you have **_any_ questions**, ***please ask***: <br />
 
 Start with a few definitions:
 
-+ **M**odel - or "data model" is the place where all data
-is often referred to as the application's `state`.
++ **M**odel - or "data model" is the place where all data stored;
+often referred to as the application's `state`.
 + **U**pdate - how the app handles `actions` performed
 by people and `update` the `state`.
-+ **V**iew - what the people using the app can _see_;
-a way to `view` the Model (counter) as `HTML` 
-rendered by the web browser.
++ **V**iew - what people using the app can _see_;
+a way to `view` the Model (counter) as `HTML`
+rendered in a web browser.
 
 ![elm-muv-architecture-diagram](https://cloud.githubusercontent.com/assets/194400/25773775/b6a4b850-327b-11e7-9857-79b6972b49c3.png)
 
-Don't worry if you don't understand this diagram (_yet_),
+<br />
+If you're not into flow diagrams, don't worry, there not everyone is, <br />
+a _much_ more "user friendly" _explanation_
+of **The Elm Architecture** ("TEA") <br />
+is
+[**Andrey Kuzmin**'s](https://github.com/w0rm) _fantastic_
```

Comment:
This is not my illustration and should not mention my name, Kolja Wilcke https://twitter.com/01k made it.

Reply:
@w0rm got it. thanks for clarifying. fixed. 👍